src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-8130.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2022-11-15'
 version: 8.13.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.13.0.gem'
+features: ["Updated support for Redis v5.0", "Updated support for Sidekiq v7.0", "Updated Rack v3.0 instrumentation to support the use of a block with Rack::Builder#run"]
+bugs: ["Correctly identify Unicorn, Rainbows, and FastCGI with Rack v3.0", "'add_method_tracer' correctly records code level metric attributes on private methods"]
```

Comment:
One more bug: 
* Add :category as a required keyword arg for NewRelic::Agent::Tracer.in_transaction